mods/tnt/init.lua

```diff
@@ -538,7 +541,7 @@ function tnt.register_tnt(def)
 			description = def.description,
 			tiles = {tnt_top, tnt_bottom, tnt_side},
 			is_ground_content = false,
-			groups = {dig_immediate = 2, mesecon = 2, tnt = 1},
+			groups = {dig_immediate = 2, mesecon = 2, tnt = 1, flammable = 5},
```

Comment:
SInce it's already burning best not make it flammable, the flame will hover above and not look good anyway.